cerealize/test/impl.cc

```diff
@@ -93,7 +96,7 @@ TEST(Cerealize, NamedJSON) {
 };
 
 TEST(Cerealize, BinarySerializesAndParses) {
-  FileSystem::CreateDirectory(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirectoryParameters::Silent);
+  FileSystem::CreateDir(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirParameters::Silent);
```

Comment:
Oh I'm sorry, I understand. `#define`s are evil.

That's the joke that lingers in my team if someone shortens words in a variable or function name and spoils readability.
> Ken Thompson was once asked what he would do differently if he were redesigning the UNIX system. His reply: "I'd spell creat with an e."
> http://en.wikiquote.org/wiki/Kenneth_Thompson